ansible_collections/arista/avd/docs/release-notes/3.x.x.md

```diff
@@ -6,13 +6,26 @@
 * Documentation for AVD version `3.4.0` [available here](https://www.avd.sh/en/v3.4.0/)
 * Documentation for AVD version <= `3.3.3` [available here](https://www.avd.sh/en/releases-v3.x.x/)
 
+## Release 3.8.0
+
+### Changes to requirements
+
+- AVD now requires ansible-core from __2.11.3__ to __2.12.x__ excluding __2.12.0__ to __2.12.5__<br>
+  Excluded versions have an [issue](## "plugin loader will now load config data for plugin by name instead of by file to avoid issues with the same file being loaded under different names (fqcn + short name).")
```

Comment:
```suggestion
  Excluded versions have an [issue](## "plugin loader will now load config data for plugin by name instead of by file to avoid issues with the same file being loaded under different names ([FQCN](https://docs.ansible.com/ansible/latest/reference_appendices/glossary.html#term-Fully-Qualified-Collection-Name-FQCN) + short name).")
```